src/questions/introduction_to_defi.yaml

```diff
@@ -0,0 +1,1147 @@
+  answerKeys:
+    - C
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "Medium"
+  choices:
+    - content: DeFi is accessible from anywhere.
+      key: A
+      order: 0
+    - content: DeFi is free to use for everyone.
+      key: B
+      order: 1
+    - content: DeFi applications are interconnected.
+      key: C
+      order: 2
+    - content: None of the above
+      key: D
+      order: 3
+- uuid: b7ffffa0-4478-40fb-8fbf-32463792876a
+  type: SingleChoice
+  content: How does DeFi solve the volatility of cryptocurrency?
+  hint: Uses of DeFi
+  explanation: Stablecoins do not fluctuate because they are backed by a highly stable asset such as gold or USD.
+  answerKeys:
+    - B
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "Low"
+  choices:
+    - content: Using smart contracts
+      key: A
+      order: 0
+    - content: Using Stablecoins
+      key: B
+      order: 1
+    - content: Using Flash Loans
+      key: C
+      order: 2
+    - content: None of these
+      key: D
+      order: 3
+- uuid: a63ed714-02dc-4f2c-956b-8dac94e583f9
+  type: SingleChoice
+  content: Centralised Finance provides a transparent system to its customers.
+  hint: Disadvantages of Centralised system
+  explanation: Centralised finance hides its functioning and procedures from its users which creates confusion and many times hidden charges for them also.
+  answerKeys:
+    - B
+  subTopics:
+    - "centralised-vs-decentralised-finance"
+  difficultyLevel: "Low"
+  choices:
+    - content: True
+      key: A
+      order: 0
+    - content: False
+      key: B
+      order: 1
+- uuid: 2c93b4b7-d351-4a32-aaaa-5170524b04dc
+  type: MultipleChoice
+  content: How does DeFi help young entrepreneurs?
+  hint: DeFi has no hidden charges and it is fast technology.
+  explanation: In DeFi there are no charges on a credit card swipe, the fund transfer is fast and affordable and allows flash loans to its users as well.
+  answerKeys:
+    - A
+    - B
+    - C
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "Medium"
+  choices:
+    - content: No credit card swipe charges
+      key: A
+      order: 0
+    - content: Fast and affordable money transfer
+      key: B
+      order: 1
+    - content: Flash Loans
+      key: C
+      order: 2
+    - content: Easy money-making opportunity
+      key: D
+      order: 3
+- uuid: 590b7500-5f8f-4f3b-8b4c-74a34ae88b95
+  type: MultipleChoice
+  content: Which of the following problems related to bank loans does DeFi solve?
+  hint: Benefits of DeFi over CeFi
+  explanation: CeFi requires a lot of collateral before the loan is granted and loans are granted only to potential borrowers with high net worth.
+  answerKeys:
+    - A
+    - C
+  subTopics:
+    - "centralised-vs-decentralised-finance"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Centralised loans require a lot of collateral
```

Comment:
`Centralised loans require a lot of collateral` Not sure if this is a correct choice as CeFi can be under-collateralized but DeFi so far is over collateralized

Reply:
made a better option